shopping-cart/src/main/java/com/example/shoppingcart/impl/ShoppingCartModule.java

```diff
@@ -12,6 +12,5 @@
     @Override
     protected void configure() {
         bindService(ShoppingCartService.class, ShoppingCartServiceImpl.class);
-        bind(ClusterBootstrapStart.class).asEagerSingleton();
```

Comment:
Wasn't the `-java-` sample app starting anything else? Weird. I think the `-scala-` also started `AkkaManagement` but I guess it happens transitively.

Reply:
I don't know if I get your point. 

`ClusterBootstrapStart` was a custom component that was eargely starting akka-management and cluster bootstrap. This is not needed in LAgom 1.5.0, so I remove it.

Reply:
I confused `ClusterBootstrapStart` with [`ClusterBootstrat().start()`+`AkkaManagement.start()`](https://github.com/lagom/shopping-cart-scala/pull/17/files#diff-1cddafe3b105a076fbc4b457f7a0b6a7L20) so I didn't understand why we _only_ needed to remove one line of code.

:-)